src/server/protocol.d.ts

```diff
@@ -304,6 +304,11 @@ declare namespace ts.server.protocol {
           * True if reference is a write location, false otherwise.
           */
         isWriteAccess: boolean;
+
+        /**
+         * True if reference is a definition, false otherwise.

```

Comment:
I thought it was necessary for API additions. I'll make it required if that's not the case.